+exp/ExpTest.m

```diff
@@ -433,7 +433,7 @@ function checkForUpdate(~)
           continue;
         else
           [~, curBranch] = ...
-            system([gitexepath, ' rev-parse --abbrev-ref HEAD'], '-echo');
+            system([gitexepath, ' rev-parse --abbrev-ref HEAD']);
```

Comment:
yeah, we talked about this earlier in https://github.com/cortex-lab/signals/pull/10/files :

> k1o0 27 days ago  Member
Giving the user the option to decline is a good idea. +git/update.m could be updated to allow this. Couldn't you just call update and check all modules? It doesn't cost much time and it's potentially important to do so as often updating alyx-matlab and not Rigbox or visa-versa causes problems. In any case we should be pushing the latest submodule commits anyway so you shouldn't need to individually check the submodules. My point, I suppose, is that the function of checkForUpdate is practically the same as update. It would be better to have just one function for updating the repo, which can be called by anyone or thing that wishes to run updates. That way you reduce code and potential errors, plus you only have to make changes to one file.

>  @jaib1
jaib1 24 days ago  Author Member
ok, I can make the necessary changes to 'git.update' and write a test function for it in the future and call it from 'exp.ExpTest' at that time

as I keep iteratively (probably in the next pull request) working on this branch, I'll do this ^

Reply:
Related to this is issue https://github.com/cortex-lab/Rigbox/issues/172